storage/src/system/auction/detail.rs

```diff
@@ -1202,15 +1285,15 @@ where
 pub fn delegators<P>(
     provider: &mut P,
     validator_public_key: &PublicKey,
-) -> Result<Vec<Box<Delegator>>, Error>
+) -> Result<Vec<Box<DelegatorBid>>, Error>
 where
     P: RuntimeProvider + ?Sized + StorageProvider,
 {
     let mut ret = vec![];
     let bid_addr = BidAddr::from(validator_public_key.clone());
     let delegator_bid_keys = provider.get_keys_by_prefix(
         &bid_addr
-            .delegators_prefix()
+            .delegated_account_prefix()
```

Comment:
This would only give us the accounts delegated the to the imputed validator right? Is there a reason we don't consider purses here as well?

Reply:
good catch, fixed on next push